workloads/bit/hpl/launch_hpl_experiment.sh

```diff
@@ -0,0 +1,500 @@
+#!/bin/bash
+#
+# Launch a HPL Burn-in test based.
+#
+# ./launch_experiment_slurm.sh <nodes per job>
+#
+# - Right now systems supported are dgx1v_16G, dgx1v_32G, dgx2, and dgxa100
+# - Eventually the code should (somewhat) support generic systems.
+#
+# -- Requirements
+# - OpenMPI 4.0.4 and UCX 1.9 must be on the PATH and LD_LIBRARY_PATH
+# - The correct cuda version should be installed and on the PATH and LD_LIBRARY_PATH
+# - run the install_hplbit_deps.sh script to do the above.
+# - Slurm Cluster Manager setup with PMIx and hwloc is required.
+#  
+#
+
+### TODO:
+### - Add a loop at the end that tracks the jobs
+###   <TIME> TotalJobs:<TOTALJOBS> RunningJobs:<RunningJobs> CompletedJobs:<CompJobs> QueuedJobs:<Queued>
+###
+###    When all the jobs are down, run the verify script, put the results in the results directory
+### - Set the full expdir from this script
+
+export HPL_DIR=${HPL_DIR:-$(cd $(dirname $0) && pwd)} # The shared directory where scripts, data, and results are stored
+export HPL_SCRIPTS_DIR=${HPL_SCRIPTS_DIR:-${HPL_DIR}} # The shared directory where these scripts are stored
+
+## Set default options
+niters=5
+partition="batch"
+usegres=1
+hplai=0
+maxnodes=9999
+grestype=""
+mpiopts=""
+walltime=1:00:00
+verbose=0
+nores=0
+cruntime=singularity
+container="nvcr.io/nvidia/hpc-benchmarks:20.10-hpl"
```

Comment:
This needs to be updated to latest HPL container: "nvcr.io/nvidia/hpc-benchmarks:21.4-hpl"